### PR TITLE
Tweak blog

### PR DESCRIPTION
**Pull request by RonaldDijkstra**
### Goals: 

- More responsive
- Less code

### In this PR: 

- New pagination style (more elegant): 

#### From: 

blog - defacto 2

#### To: 

blog - defacto 3

Doesn't hide the next and previous step anymore on mobile browsers (cool)

- Use flex box for article previews
- Make the post layout more responsive (no jumps when resizing the page)
- Use a bit less spacing, in the blog-headers for example
- Make the post meta stuff (author, date, tags) look and act the same everywhere

Tested in IE10+, Edge, FF, Chrome, Safari, Chrome Mobile, Safari Mobile

### CODE REVIEW

File: source/blog/_blog-article-meta.erb [hunk 1]
```diff
@@ -9,9 +9,11 @@ author_name = author ? blog_author_name(author) : article.data.author
     <%= I18n.l(article.date, format: :blog_date) %>
   </time>
   <span class="post-tags">
-    <% article.tags.each do |tag| %>
+    <% article.tags.each_with_index do |tag, index| %>
       <%= link_to capitalize(tag), tag_path(tag, "blog"), class: "tag" %>
-      <span class="separator">&bull;</span>
+      <% unless index == article.tags.size - 1 %>
```
Comment (RonaldDijkstra) [inline source/blog/_blog-article-meta.erb]: The best way to exclude the last seperator? It was `display: none` with css before

Comment (fatboypunk) [inline source/blog/_blog-article-meta.erb]: and why is that removed?

File: source/fonts/selection.json [hunk 2]
```diff
@@ -475,4 +529,4 @@
     "showCodes": true,
     "gridSize": 16
   }
-}
+}
```
Comment (fatboypunk) [inline source/fonts/selection.json]: end of file!

**Comment by fatboypunk** [comment]
@fritsypeij Can you take a look at this?

File: source/stylesheets/_blog.scss [hunk 3]
```diff
@@ -13,52 +10,57 @@
...
+        display: inline-block;
+      }
+
       &::before,
       &::after {
-        color: $post-meta;
-        content: "–";
-        display: inline;
-        margin: 0 0.5em;
+        display: none;
```
Comment (fritsypeij) [inline source/stylesheets/_blog.scss]: I think there's no need for this

File: source/stylesheets/_blog.scss [hunk 4]
```diff
@@ -256,19 +151,93 @@
...
+          line-height: 3em;
+          margin-left: -1px;
+          padding: 0 1em;
+          text-decoration: none;
+
+          @include media($medium-up) {
+            font-size: $base-font-size;
+          }
+
+          &:active,
```
Comment (fritsypeij) [inline source/stylesheets/_blog.scss]: you can put the li outside of the ul (a li is always nested in ul anyway)

File: source/blog/_pagination.erb [hunk 5]
```diff
@@ -1,25 +1,25 @@
...
 
       <% for page in 1..num_pages %>
+        <%
+        page_url = "/blog/"
+        page_url << "tags/#{tagname.downcase}/" if defined?(tagname)
+        page_url << "#{page}/" unless page == 1
+        %>
         <% if page == page_number %>
-          <li class="current"><%= page %></li>
+          <li class="current"><%= link_to page, page_url %></li>
```
Comment (fritsypeij) [inline source/blog/_pagination.erb]: Doesn't have to be a link, maybe make it a span?

File: source/stylesheets/_blog.scss (hunk 4 above)
Comment (RonaldDijkstra) [inline source/stylesheets/_blog.scss]: Updated the nesting to max 6 earlier, but I get the point

File: source/stylesheets/_blog.scss (hunk 3 above)
Comment (RonaldDijkstra) [inline source/stylesheets/_blog.scss]: Oh right, below medium-up it doesn't have content anyway

File: source/blog/_blog-article-meta.erb (hunk 1 above)
Comment (RonaldDijkstra) [inline source/blog/_blog-article-meta.erb]: Because I think this is a better approach for not showing an item than through styling.
